docs/team/ian-from-dover.md

```diff
@@ -5,43 +5,53 @@ title: Ian Hong's Project Portfolio Page
 
 ### Project: PeopleSoft
 
-PeopleSoft lets HR managers of companies handle their people.
+PeopleSoft is a Payroll management app for companies offering contractor-based services.
 
 
-Given below are not my contributions to the project. (copied over)
+### Summary of Contributions
+
+* **Role in team**: Team leader and frontend developer
+* **New Feature**: Added the ability to open the User Guide directly in the web browser. (Pull Request [\#77](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/77))
+  * **What it does**: allows the user to open the User Guide directly in the web browser.
```

Comment:
Just a small nit, `allows` should probably be capitalized. Otherwise, this looks good.